docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -329,7 +333,7 @@ public class OidcTokenRequestCustomizer implements OidcRequestFilter {
 <1> Get `OidcConfigurationMetadata` which contains all supported OIDC endpoint addresses.
 <2> Use `OidcConfigurationMetadata` to filter requests to the OIDC token endpoint only.
 
-Alternatively, you can use `OidcRequestFilter.Endpoint` enum to make sure this filter is applied to the token endpoint requests only:
+Alternatively, you can use `OidcRequestFilter.Endpoint` enum to verify this filter is applied to the token endpoint requests only:
```

Comment:
```suggestion
Alternatively, you can use `OidcRequestFilter.Endpoint` enum to have this filter applied to the token endpoint requests only:
```

`verify` does not work

Reply:
Thanks @sberyozkin:
Propose an additional slight change to avoid the passive voice:
"Alternatively, you can use `OidcRequestFilter.Endpoint` enum to apply this filter to the token endpoint requests only:"

Reply:
Updated to "Alternatively, you can use OidcRequestFilter.Endpoint enum to apply this filter to the token endpoint requests only:"